### PR TITLE
Update of syntax and indentation

### PR DESCRIPTION
**Pull request by DamCorreia**
Update help tags as param tags attribute
Clean labels

### CODE REVIEW

File: tools/mrbayes/mrbayes.xml [hunk 1]
```diff
@@ -163,45 +175,52 @@ See the manual_ or the `command reference`_ for more information.
...
- GRT: lset nst=6;
- SYM: lset nst=6; prset statefreqpr=fixed(equal);
- HKY: lset nst=2;
- K2P: lset nst=2; prset statefreqpr=fixed(equal);
- F81: lset nst=1;
- JC: lset nst=1; prset statefreqpr=fixed(equal);
- Mixed: lset nst=mixed;
+GRT: lset nst=6;
+SYM: lset nst=6; prset statefreqpr=fixed(equal);
+HKY: lset nst=2;
```
Comment (ericenns) [inline tools/mrbayes/mrbayes.xml]: Should HKY be updated to HKY85

File: tools/mrbayes/mrbayes.xml [hunk 2]
```diff
@@ -163,45 +175,52 @@ See the manual_ or the `command reference`_ for more information.
...
+K2P: lset nst=2; prset statefreqpr=fixed(equal);
+F81: lset nst=1;
+JC: lset nst=1; prset statefreqpr=fixed(equal);
```
Comment (ericenns) [inline tools/mrbayes/mrbayes.xml]: Should JC be updated to JC69

**Comment by ericenns** [comment]
Are you planning on any more changes? or should we be pushing a new version?

**Comment by DamCorreia** [comment]
Yes, i am but not immediatly, so you could push this version. Thank you very much for your collaboration

**Pull request merged**